DB/DBI.pm

```diff
@@ -63,7 +63,7 @@ sub getConfigSetting
     $where = " WHERE c.ConfigID=(Select cs.ID from ConfigSettings cs where cs.Name=?)";
     $query = " SELECT c.Value FROM Config c";
     $query = $query . $where;
-    my $sth = $$dbh->prepare($query);
+    my $sth = $dbh->prepare($query);
```

Comment:
done! I changed it because it was not working for @gluneau 
(both ways work for me!)